HPXMLtoOpenStudio/resources/data/unavailable_periods.csv

```diff
@@ -36,5 +36,5 @@ bath_fan,1,1,0,0
 whole_house_fan,0,1,0,0
```

Comment:
I agree with your change. Looking at the other end uses, it also seems like whole house fans should be unavailable during vacancies. Everything else looks reasonable to me.

```suggestion
whole_house_fan,1,1,0,0
```